pkg/image/source.go

```diff
@@ -125,6 +131,7 @@ func detectSource(fs afero.Fs, userInput string) (Source, string, error) {
 					// the docker daemon exists and is accessible
 					return DockerDaemonSource, userInput, nil
```

Comment:
This entire `case UnknownSource: ...` should probably get updated to ping podman if docker can't be found and fallback to returning `PodmanDaemonSource`.